.github/.github/workflows/workflow

```diff
@@ -0,0 +1,48 @@
+name: Linters
```

Comment:
- [ ] I would kindly suggest you to properly set the location of `linters.yml` file and its content. Currently, in your project linters does not work. To do this you can follow 3 easy steps: 

1. Create `.github` folder 
2. Inside .`github` folder create `workflows` folder
3. Inside `workflows` folder add `linters.yml` file with [this](https://github.com/microverseinc/linters-config/blob/master/html-css/.github/workflows/linters.yml) content 

- [ ] Additionally, to make linters work, please add `.hintrc` file and `.stylelintrc.json` file into your project. Kindly copy above mentioned files from [this](https://github.com/microverseinc/linters-config/tree/master/html-css) repository 

Please feel follow the [link](https://github.com/microverseinc/linters-config/tree/master/html-css) on linters setup and make sure to go through all the steps.